build/template-install-dotnet-core.yaml

```diff
@@ -5,6 +5,6 @@ steps:
 - task: UseDotNet@2
   displayName: 'Use .Net Core SDK'
   inputs:
-    version: 3.1.101, 5.0.100-preview.5.20279.10
+    version: 5.0.100-preview.5.20279.10
```

Comment:
yeah, I was suprised that you can have a list there :)

Reply:
that's what the documentation says to do. idk. doesn't seem to work.

Reply:
@bgavrilMS had to make separate tasks for each TFW